hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -5,15 +5,13 @@ Prevent Popups to overflow tables.
 */
 (function ($) {
   'use strict';
+  var $body = $('body');
 
-  $('body').on('click', '.btn-slide', function (e) {
+  $('body').on('click', '.btn-slide', function () {
     var $button = $(this);
-    var $body = $('body');
     var $closestEntity = $button.closest('.crm-entity')[0];
     var $popup = $button.children('ul.panel');
 
-    // e.preventDefault();
```

Comment:
Not sure how I didn't notice that this was commented, probably was looking in a different commit. I wonder if we should call e.stopPropagation() in case buttons appear inside other blocks with click events?